### DISCUSSION TITLE
Weird looking graphics if you haven't watched any Courses yet

### DISCUSSION THREAD

**Discussion by josiebytes** [Bootstrap-Academy/Bootstrap-Academy#56] [Polls]
### Bootstrap Academy username

Joshiy13

## Description

If you go to your profile page and you haven't watched any courses on that account, it shows empty boxes (Probably thumbsnails).

## Fix

Remove the empty boxes/thumbsnails from the page if the user hasn't watched any videos yet.

## Screenshots

![image](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/79088559/a4e22fd2-79bb-4a6b-a473-e41cec6e9dea)

**Comment by vimgrund** [comment]
this seems to be by design, see https://github.com/Bootstrap-Academy/frontend/blob/develop/components/course/CardSmEmptyState.vue#L5

to remove it, you can remove this line 
https://github.com/Bootstrap-Academy/frontend/blob/develop/components/user/Courses.vue#L37
![grafik](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/75255380/56afd1a3-9ff0-4ce8-9d77-c1d2b76eee8e)

but it doesn't really look "nicer" in my opinion
![grafik](https://github.com/Bootstrap-Academy/Bootstrap-Academy/assets/75255380/46215e26-fd4c-4dcb-946e-525f3ec3d591)

**Comment by josiebytes** [comment]
I feel like it would look nicer if there was emtpy space instead of the thumbnails

**Comment by TheMorpheus407** [comment]
Let's make a vote :)
Having empty cards is better

**Comment by TheMorpheus407** [comment]
Having empty space with no cards is better
